config/initializers/datadog.rb

```diff
@@ -1,23 +1,9 @@
 Datadog.configure do |c|
-  c.tracer env: Rails.env
   c.tracer enabled: ENV["DD_API_KEY"].present?
   c.tracer partial_flush: true
-  c.tracer priority_sampling: true
-
-  c.use :concurrent_ruby
-  c.use :excon, split_by_domain: true
-  c.use :faraday, split_by_domain: true
-  c.use :http, split_by_domain: false
-  c.use :httpclient, split_by_domain: false
-  c.use :httprb, split_by_domain: true
-  c.use :rails
-
-  c.use :rest_client
-  c.use :sidekiq
```

Comment:
With `ddtrace/auto_instrument` in place, there's no need to verbosely declare what library to instrument. By default the following are automatically enabled/selected
```
$ Datadog.configuration.instrumented_integrations.keys
=> [:active_model_serializers, :aws, :concurrent_ruby, :dalli, :delayed_job, :elasticsearch, :ethon, :excon, :faraday, :grape, :graphql, :grpc, :http, :httpclient, :httprb, :kafka, :mongo, :mysql2, :presto, :qless, :que, :racecar, :rails, :rake, :redis, :resque, :rest_client, :sequel, :shoryuken, :sidekiq, :sinatra, :sneakers, :sucker_punch, :rack, :action_mailer, :active_support, :action_pack, :action_view, :active_job, :active_record]
```


Regarding `split_by_domain: true`, I'm having a hard time justifying it's purpose. Current it seems to give us a really granular data on how we are interacting with a particular service (ie. github api, mailchimp api) which isn't very useful. I'm removing it for the time being.